solutionbox/structured_data/datalab_solutions/structured_data/_package.py

```diff
@@ -230,16 +222,17 @@ def local_train(preprocessed_dir, transforms_config_file, output_dir,
   print('Local training done.')
 
 
-def cloud_train(preprocessed_dir, transforms_config_file, output_dir,
-                staging_bucket,
+def cloud_train(preprocessed_dir, schema_file, transforms_file, output_dir,
+                model_type, staging_bucket,
```

Comment:
Very true, but for dataflow, the stage location can be any gcs path (gs://bucket/my/path/tmp/stage), So it's easy to say that we can make the stage location just "output_dir/tmp/stage".

However, for training (via gcloud?) the stage location has to be the root bucket (so gs://bucket/my/path/tmp/stage does not work). Once they fix that issue, I want to move over to just using the output folder.